.github/workflows/dab-consumers.yml

```diff
@@ -28,6 +28,7 @@ jobs:
           FROM ghcr.io/ansible/awx_devel:devel
           COPY . /opt/dab/
           RUN pip install -e /opt/dab/
+          RUN pip install jq
```

Comment:
Bugs me that this was merged still despite it not working, reason it did not work though is because it probably sources a venv to run the tests. Not sure when the venv is created and installs its dependencies though.

https://github.com/ansible/django-ansible-base/actions/runs/13504381487/job/37730418006#step:6:41

Reply:
Oh in that case i probably messed up in tab management. Sorry about this one.